elk/training/train.py

```diff
@@ -43,20 +44,22 @@ class RunConfig(Serializable):
     max_gpus: int = -1
     normalization: Literal["legacy", "none", "elementwise", "meanonly"] = "meanonly"
     skip_baseline: bool = False
+    concatenate_layers: int = 0
```

Comment:
oh this is tricky, I feel like we can come up with a better name for this but I'm not sure what it should be atm...